packages/components/src/components/lineage/types.ts

```diff
@@ -56,6 +56,7 @@ export interface LineageOptions {
     filterIn?: boolean;
     filters?: LineageFilter[];
     grouping?: LineageGroupingOptions;
+    // the Map<string, string> should be keyed off of the queryName for the group show title suffix is being modified
```

Comment:
Not sure I follow this comment. Is the map a way to modify the group title suffix for a specific queryName? If so, does this make sense:
```suggestion
    // the Map<string, string> should be keyed off of the queryName for the group title suffix being modified
```

Reply:
Yes, thanks.  That "show" was meant to be "whose" but your wording is more succinct.